artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -165,7 +165,7 @@
 
    private volatile boolean queueDestroyed = false;
 
-   private final PageSubscription pageSubscription;
+   protected final PageSubscription pageSubscription;
```

Comment:
Is this needed?

Reply:
These access changes were left-over from when RingQueue was subclassed. I'll fix them.